css_tasks/task3/main.css

```diff
@@ -0,0 +1,45 @@
+*{
+    margin:0;
```

Comment:
generally it is a bit easier to read when you have space after selector and space before property value

```
selector {
  margin: 0;
}
```

Reply:
Also always try to follow some properties order when write styles, you need to have some logic behind why your properties in some order [link](https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685)